### PR TITLE
Bump NodeJS versions to 10.15.1 lts

### PR DESCRIPTION
**Pull request by davidpmccormick**
Builds successfully and runs locally without any apparent issues on `10.15.1`. Not keen to deploy until @jamesgorrie has been over it, though.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -209,7 +209,7 @@ jobs:
             fi
   edge_lambdas_build:
     docker:
-      - image: circleci/node:8.11.3
```
Comment (davidpmccormick) [inline .circleci/config.yml]: Wondering if we should pin this to 8.10 instead?

> AWS Lambda supports the following Node.js runtimes.
> 
> Node.js Runtimes
> 
> Name | Identifier
> -- | --
> Node.js 8.10 | nodejs8.10
> Node.js 6.10 | nodejs6.10
> 
> 

And would be in line with [what we have here](https://github.com/wellcometrust/wellcomecollection.org/blob/master/cache/edge_lambdas.tf#L29)

File: build/Dockerfile_webapp [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:8.11.3-alpine
+FROM node:10.15.1-alpine
 
 ARG WEBAPP_NAME
 ARG WEBAPP_PORT
```
Comment (jamesgorrie) [inline build/Dockerfile_webapp]: We can kill this file, but maybe we remove all the `./build` stuff in a separate PR.

File: .circleci/config.yml (hunk 1 above)
Comment (jamesgorrie) [reply .circleci/config.yml]: As it's where the tests are running, this makes loads of sense to stop us using features that aren't available in `8.10`.

**Comment by jamesgorrie** [comment]
Just wondering, should we [set the node version in the `package.json`s](https://docs.npmjs.com/files/package.json#engines) to make sure we're running the same version locally?

I know it will run with a lower version as we're not using features from the new one, but creates less disparity between local & prod.

File: .circleci/config.yml (hunk 1 above)
Comment (jamesgorrie) [reply .circleci/config.yml]: To be less opaque:

> Wondering if we should pin this to 8.10 instead?

Yip! Great idea.

**Pull request merged**